docs/Windows.md

```diff
@@ -3,25 +3,55 @@
+  - [Multiple definition of `cleanup_glue`, `_reclaim_reent` etc](#multiple-definition-of-cleanup_glue-_reclaim_reent-etc)
+
+To build using Visual Studio [see here](Windows-VisualStudio.md). The rest of these instructions cover command-line builds.
 
 ## Prerequisites
 
-You'll need to install:
+- Windows Terminal - grab it from the Windows Store
+- [Git for Windows](https://git-scm.com/download/win)
+- [Python](https://www.python.org/downloads/) - eg: "python-3.9.6-amd64.exe"
+- The [GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) (For 32blit device builds, use the 9-2020-q2-update version). Make sure to select the "Add path to environment variable" at the end of the setup.
+ - The [Visual Studio build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) (For local builds, select "C++ build tools"). This includes CMake and NMake.
+
+If you have Visual Studio installed you can Modify the installation and select "C++/CLI support for vXXX build tools (Latest)"
+
+## Setting Up
+
+*Note: Developer Command Prompt is a little clunky but you can add it to Windows Terminal with the following config:*
+
+```json
+{       
+    "commandline": "cmd.exe /k \"C://Program Files (x86)//Microsoft Visual Studio//2019//Community//Common7//Tools//VsDevCmd.bat\"",
+    "cursorColor": "#EEEEEE",
+    "cursorShape": "bar",
+    "fontFace": "Consolas",
+    "fontSize": 10,
+    "guid": "{5ee0706e-b015-46b2-98a3-2122a8e627d3}",
+    "historySize": 9001,
+    "icon": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\Assets\\VisualStudio.70x70.contrast-standard_scale-80.png",
+    "name": "Developer Command Prompt for VS2019",
+    "padding": "0, 0, 0, 0",
+    "snapOnInput": true,
+    "startingDirectory": "%USERPROFILE%"
+}
+```
+
+*Modified from this blog post: https://yer.ac/blog/2021/03/09/adding-vs-developer-command-prompt-to-windows-terminal-vs-2019/*
+
+Open "Developer Command Prompt for VS 2019." This is where you'll build the 32blit SDK and flash .blit files to your device.
 
- - [Git for Windows](https://git-scm.com/download/win)
- - [Python](https://www.python.org/downloads/)
- - At least the [Visual Studio build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) (For local builds, select "C++ build tools"). This includes a copy of CMake. A full Visual Studio install can also be used.
- - The [GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) (For 32blit device builds, use the 9-2020-q2-update version). Make sure to select the "Add path to environment variable" at the end of the setup.
+Install the 32blit tools:
 
-Now open "Developer Command Prompt for VS 2019" and install the 32blit tools:
 ```
-py -m pip install 32blit
+pip install 32blit
```

Comment:
My python install (3.10.2) doesn't seem to have `pip` (or `python`) only `py`.

Reply:
This is still true, should drop this change too...
<img width="867" alt="snakes" src="https://github.com/32blit/32blit-sdk/assets/3074891/6b57a907-afbd-49d2-8825-ed2f79c262c0">